projects/packages/my-jetpack/_inc/data/products/use-activate.ts

```diff
@@ -1,11 +1,24 @@
 import { __, sprintf } from '@wordpress/i18n';
+import useAnalytics from '../../hooks/use-analytics';
 import { REST_API_SITE_PRODUCTS_ENDPOINT } from '../constants';
 import { QUERY_ACTIVATE_PRODUCT_KEY } from '../constants';
 import useSimpleMutation from '../use-simple-mutation';
 import useProduct from './use-product';
+import type { ProductCamelCase } from '../types';
+
+const getIsPluginAlreadyActive = ( detail: ProductCamelCase ) => {
```

Comment:
Thinking about potentially removing this and only calling the `activate` mutation when a product is not already activated 🤔 Though that does handle the installation too. Not sure yet